sympy/integrals/tests/test_meijerint.py

```diff
@@ -624,11 +624,11 @@ def test_messy():
                        fourier_transform)
     assert laplace_transform(Si(x), x, s) == ((-atan(s) + pi/2)/s, 0, True)
 
-    assert laplace_transform(Shi(x), x, s) == (acoth(s)/s, 1, s > 1)
+    assert laplace_transform(Shi(x), x, s) == (acoth(s)/s, 1, (s > -oo) & (s < -1))
```

Comment:
Are we sure what the result should be here and below? I also made some changes that affected these, but didn't really come to any conclusion what it actually should be.

Reply:
`acoth(x)/x` is symmetric above 1 and below -1 so I think this is fine.

Reply:
So it should really say `abs(s) > 1`? (Not that it should be fixed in the PR.)